src/rust/core/server/src/threads/workers/multi.rs

```diff
@@ -118,6 +118,9 @@ where
             }
 
             WORKER_EVENT_TOTAL.add(events.iter().count() as _);
+            if events.iter().count() == self.nevent {
+                WORKER_EVENT_MAX.increment();
```

Comment:
`WORKER_EVENT_MAX` is ambiguous because it could be interpreted as a gauge value on the max number of events returned in one `poll`. To disambiguate, I suggest renaming this to `WORKER_EVENT_MAX_REACHED` or something to that effect.

Reply:
done